tests/test_widgets.py

```diff
@@ -588,7 +588,7 @@ def test_add_card_widget_ke_chain_pages(self):
 
         self.assertEqual(
             len(self.wm),
-            9,
+            13,
```

Comment:
Can we alter tge test to avoid len?

Reply:
No,  this is Jelle's legacy and I am not touching it